README.md

```diff
@@ -108,6 +108,19 @@ Note that the roadmap outlined above is subject to change based on user feedback
 This app uses Webpack (via Webpacker) and Babel in order to make use of modern JS, so don't forget to run `bin/webpack-dev-server` when developing locally.
 If you're deploying to Heroku, ensure you have the `heroku/nodejs` and `heroku/ruby` buildpacks configured (in that order).
 
+## Storybook
+
+run storybook in development
+```
+npm run storybook
+```
+
+build static storybook site
+```
+npm run build-storybook -- -o out7650
```

Comment:
What's the `out7650` mean?

Reply:
this is the name of the folder that will be generated. it contains the static site for storybook